dev/doc/build-system.dune.md

```diff
@@ -18,31 +22,26 @@ call `make -f Makefile.dune voboot`. It is usually enough to do that
 once per-session.
 
 More helper targets are availabe in `Makefile.dune`, `make -f
-Makefile.dune` will display help.
+Makefile.dune` will display some help.
 
-Dune
-====
+Dune places build artifacts in a separate directory `_build`; it will
+also generate an `.install` file so files can be properly installed by
+package managers.
 
-Coq can now be built using
-[Dune](https://github.com/ocaml/dune). Contrary to other systems,
-Dune, doesn't use a global`makefile` but local build files named
-`dune` that are later composed to form a global build.
+Contrary to other systems, Dune, doesn't use a global`makefile` but
```

Comment:
Missing space and capital letter for Makefile.

Reply:
Fixed thanks. [Note that `make` does accept both `Makefile` and `makefile`]